### ISSUE TITLE
CKEditorError: ckeditor-duplicated-modules

### ISSUE THREAD

**Issue by TaoSun01** [ckeditor/ckeditor5#16356]
import Plugin from "@ckeditor/ckeditor5-core/src/plugin"; 
import ButtonView from "@ckeditor/ckeditor5-ui/src/button/buttonview";
as soon as I introduce these two, an error will be reported

master file
```javascript
<template>
  <div>
    <ckeditor
      :editor="editor"
      v-model="editorData"
      :config="editorConfig"
    ></ckeditor>
  </div>
</template>

<script>
import ClassicEditor from "@ckeditor/ckeditor5-build-classic";
import CKEditor from "@ckeditor/ckeditor5-vue2";
import CustomButton from "./CustomButton"; // 引入自定义按钮插件

export default {
  components: {
    ckeditor: CKEditor.component,
  },
  data() {
    return {
      editor: ClassicEditor,
      editorData: "<p>Content of the editor.</p>",
      editorConfig: {
        toolbar: [
          "heading",
          "|",
          "bold",
          "italic",
          "|",
          "bulletedList",
          "numberedList",
          "|",
          "undo",
          "redo",
          "|",
          "customButton", // 这里是自定义按钮
        ],
        extraPlugins: [CustomButton],
      },
    };
  },
};
</script>

<style>
#app {
  margin: 20px;
}
</style>
```
plug-in
```javascript
import Plugin from "@ckeditor/ckeditor5-core/src/plugin";
import ButtonView from "@ckeditor/ckeditor5-ui/src/button/buttonview";

export default class CustomButton extends Plugin {
  init() {
    const editor = this.editor;

    editor.ui.componentFactory.add("customButton", (locale) => {
      const view = new ButtonView(locale);

      view.set({
        label: "Custom Button",
        withText: true,
        tooltip: true,
      });

      // 添加按钮的点击事件
      view.on("execute", () => {
        editor.model.change((writer) => {
          const insertPosition =
            editor.model.document.selection.getFirstPosition();
          writer.insertText("Custom Text", insertPosition);
        });
      });

      return view;
    });
  }
}
```
![image](https://github.com/ckeditor/ckeditor5/assets/96215749/6ab9a865-a66e-48cb-ab92-bc7e888256fa)
![image](https://github.com/ckeditor/ckeditor5/assets/96215749/45624bc0-2d1b-4f39-9458-166e7bb0eb5e)

**Comment by polem** [comment]
I had the same issue with version 41.2.0, issue fixed by updating to version 42.0.0

**Comment by Reinmar** [comment]
Before v42.0.0 you could not add a plugin to an existing build (module imported from build-\* or generated using the old builder).

Starting with v42.0.0 everything has changed. As others mentioned, upgrading to v42.0.0 will resolve those issues. If you were using a version below v42.0.0 check out the migration guide: https://ckeditor.com/docs/ckeditor5/latest/updating/nim-migration/predefined-builds.html
